include/multipass/cli/client_common.h

```diff
@@ -53,6 +53,7 @@ std::unique_ptr<SSLCertProvider> get_cert_provider();
 void set_logger();
 void set_logger(multipass::logging::Level verbosity); // full param qualification makes sure msvc is happy
 void preliminary_setup();
+void final_adjustments();
```

Comment:
This one sounds weird :)

Something like `post_setup`, `finally`, `before_exit`… "adjustments" say a bit too much about what is supposed to happen inside…

Reply:
OK, sure, I'll change that.

Reply:
done